architecture/additional_concepts/dynamic_admission_controllers.adoc

```diff
@@ -0,0 +1,524 @@
+[[architecture-additional-concepts-dynamic-admission-controllers]]
+= Custom Admission Controllers
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+== Overview
+
+In addition to the the default xref://architecture/additional_concepts/admission_controllers.html#architecture-additional-concepts-admission-controllers[admission controllers], 
+you can use a custom _webhook admission controller_ as part of the admission chain. 
+
+Webhook admission controllers call webhooks to either mutate pods upon creation, such as to inject labels, 
```

Comment:
this sounds strange. Is the webhook admission controller the webhook or the native controller calling webhooks?

Technically we have this:
- `Mutating/ValidatingAdmissionWebhook` is the compiled-in admission controller that can call registered webhooks
- the webhook implements the `AdmissionReview` API and is registered via  `Mutating/ValidatingAdmissionConfiguration`.

In other words: "custom webhook admission controller" and "Webhook admission controllers" should be clear distinct. But in our wording they are nearly the same.

We could say "Webhook admission controller" for `Mutating/ValidatingAdmissionWebhook`, and "admission webhook" for the webhook implementation (with the  `AdmissionReview` API as protocol).